app/dashboard.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>CADET LMS</title>
+</head>
+
+<body>
+  <h1>Dashboard Professional</h1>
+
+  <container class="assignments"><h2>Due Right Fucking Now<h2>Show/Hide
+  <h3>Javscript to Death</h3><button>View</button>
```

Comment:
since these are placeholders it's find but I'd like to see the stuff between the containers indented another level.